### PR TITLE
Fix some packaging typos

### PR DESCRIPTION
**Pull request by borrrden**
LiteCore.framework -> LiteCore.xcframework
net6.0-android31.0 -> net6.0-android31
Condition for copying LiteCore.dll

### CODE REVIEW

File: src/Couchbase.Lite.Shared/API/Info/Defaults.cs [hunk 1]
```diff
@@ -78,10 +78,10 @@ public static class Constants {
 		public static readonly int DefaultReplicatorMaxAttemptsSingleShot = 10;
 
 		/// <summary>
-		/// Default value for <see cref="ReplicatorConfiguration.MaxAttempts" /> (-1)
+		/// Default value for <see cref="ReplicatorConfiguration.MaxAttempts" /> (Int32.MaxValue)
 		/// When replicator is continuous, never give up unless explicitly stopped
 		/// </summary>
```
Comment (bmeike) [inline src/Couchbase.Lite.Shared/API/Info/Defaults.cs:83]: I think this could be a value from Defaults

**Review by bmeike — APPROVED** [review]
How did you discover these typos?
I have the vague feeling that I've seen this before.  If that is true, how did the typos slip by?

File: src/Couchbase.Lite.Shared/API/Info/Defaults.cs (hunk 1 above)
Comment (borrrden) [reply src/Couchbase.Lite.Shared/API/Info/Defaults.cs:83]: Not sure what you mean.  It is.

**Comment by borrrden** [comment]
I bet you are remembering them from the actual project files and/or scripts that had a similar change.  The build was failing before the packaging step and in my flurry of rushing I didn't remember to test the packaging logic.  I have now done so.

**Pull request merged**